charts/posthog/templates/ingress.yaml

```diff
@@ -56,8 +56,13 @@ spec:
     - http:
   {{- end }}
         paths:
+          {{- if (eq (include "ingress.type" .) "nginx") }}
```

Comment:
Let's swap the default - `Prefix` should work in most places as that's what k8s docs have & the regex thing is different & we can't easily make assumptions about how regex needs to be provided.

My comment from slack 
for nginx we can use the regex with ".*" style (https://kubernetes.github.io/ingress-nginx/user-guide/ingress-path-matching/) if we add the annotation use-regex (https://kubernetes.github.io/ingress-nginx/user-guide/nginx-configuration/annotations/#use-regex)
but GKE expects it to be "*" (https://cloud.google.com/kubernetes-engine/docs/concepts/ingress), so looks like we need to split it based on which ingress controller we have.